lib/stream.h

```diff
@@ -123,9 +123,7 @@ struct stream_fifo {
 #define STREAM_CONCAT_REMAIN(S1, S2, size) ((size) - (S1)->endp - (S2)->endp)
 
 /* deprecated macros - do not use in new code */
-#define STREAM_PNT(S)   stream_pnt((S))
```

Comment:
1) Add code to note these are going away and should not be used
2) Mark for removal in a year

Reply:
these were marked deprecated in 2005, do we really need to do another deprecation window?

Reply:
Yep unfortunately I believe we do